crates/turborepo-ui/src/tui/table.rs

```diff
@@ -46,29 +52,39 @@ impl<'b> TaskTable<'b> {
     }
 
     fn finished_rows(&self) -> impl Iterator<Item = Row> + '_ {
-        self.tasks_by_type.finished.iter().map(move |task| {
-            let name = if matches!(task.result(), TaskResult::CacheHit) {
-                Cell::new(Text::styled(task.name(), Style::default().italic()))
-            } else {
-                Cell::new(task.name())
-            };
+        self.tasks_by_type
+            .finished
+            .iter()
+            // note we can't use the default Ord impl because
+            // we want failed tasks first
+            .sorted_by_key(|task| match task.result() {
```

Comment:
Non blocking TODO for myself (or anyone else), but we should try to insert tasks as they finish in this order instead of sorting it on every render.